runtime/lua/vim/lsp/protocol.lua

```diff
@@ -632,7 +632,16 @@ function protocol.make_client_capabilities()
 
         codeActionLiteralSupport = {
           codeActionKind = {
-            valueSet = {};
+            valueSet = {
+              protocol.CodeActionKind.Empty,
+              protocol.CodeActionKind.QuickFix,
+              protocol.CodeActionKind.Refactor,
+              protocol.CodeActionKind.RefactorExtract,
+              protocol.CodeActionKind.RefactorInline,
+              protocol.CodeActionKind.RefactorRewrite,
+              protocol.CodeActionKind.Source,
+              protocol.CodeActionKind.SourceOrganizeImports,
+            };
```

Comment:
use `vim.tbl_values` to retreive the values from CodeActionKind programmatically.

Reply:
Thanks!